packages/manager/src/components/TextTooltip/TextTooltip.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TextTooltip } from './TextTooltip';
+
+describe('TextTooltip', () => {
+  it('renders the display text and displays the tooltip on hover', async () => {
+    const props = {
+      displayText: 'Hover me',
+      tooltipText: 'This is a tooltip',
+    };
+
+    render(<TextTooltip {...props} />);
```

Comment:
What do you think about using this syntax instead of using the `screen` object. 

I don't know if this is actually true, but this feels safer to me compared to the the magical `screen` import. 

```suggestion
    const. { findByRole } = render(<TextTooltip {...props} />);
```

Reply:
I cleaned the the up for good measure 👍